yellowbrick/cluster/elbow.py

```diff
@@ -18,7 +18,7 @@
 ## Imports
 ##########################################################################
 
-import collections
+from collections.abc import Iterable
```

Comment:
Thank you so much for fixing this deprecation warning!

Reply:
I might have to revert this because `collections.abc` is not available in python 2.7.... Any thoughts?

Reply:
We're planning on deprecating Python 2.7 support in version 1.0 (#706) -- so I guess now is as good a time as any to do that! I'll get a PR open for that so that this PR doesn't break CI.